src/main/resources/interface-specification.yml

```diff
@@ -1738,8 +1738,6 @@ components:
         assignmentTimestamp:
           type: string
           format: date-time
-        renewal:
```

Comment:
[this](https://github.com/pagopa/interop-be-backend-for-frontend/pull/39/files#diff-17a3d713b849787a0ed57552f2b367a47eedf4cef81a6ba025d0060ac107d213R1753) line needs to be removed, and the `renewal` field should be added to `TenantVerifier` and `TenantRevoker`